etc/kernel-launchers/scala/toree-launcher/build.sbt

```diff
@@ -17,4 +17,4 @@ val sparkVersion = "2.4.1"
 
 libraryDependencies += "org.apache.spark" %% "spark-sql" % sparkVersion % "provided"
 libraryDependencies += "com.typesafe.play" %% "play-json" % "2.3.10" // Apache v2
-libraryDependencies += "org.apache.toree" % "toree-assembly" % "0.3.0-incubating" /*from "http://archive.apache.org/dist/incubator/toree/0.3.0-incubating/toree/toree-assembly-0.3.0-incubating.jar"*/
+libraryDependencies += "org.apache.toree" % "toree-assembly" % "0.4.0-incubating" /*from "http://archive.apache.org/dist/incubator/toree/0.4.0-incubating/toree/toree-assembly-0.4.0-incubating.jar"*/
```

Comment:
It looks like toree-0.3.0 is still used.  Is this because scalaVersion is still `2.11` above?

```
[info] Set current project to toree-launcher (in build file:/Users/kbates/repos/oss/jupyter/enterprise_gateway/etc/kernel-launchers/scala/toree-launcher/)
[info] Updating ...
[info] downloading https://repo1.maven.org/maven2/org/apache/toree/toree-assembly/0.4.0-incubating/toree-assembly-0.4.0-incubating.jar ...
[info] 	[SUCCESSFUL ] org.apache.toree#toree-assembly;0.4.0-incubating!toree-assembly.jar (1307ms)
[info] Done updating.
[warn] There may be incompatibilities among your library dependencies.
[warn] Run 'evicted' to see detailed eviction warnings
[info] Compiling 4 Scala sources to /Users/kbates/repos/oss/jupyter/enterprise_gateway/etc/kernel-launchers/scala/toree-launcher/target/scala-2.11/classes ...
[info] Done compiling.
[info] Packaging /Users/kbates/repos/oss/jupyter/enterprise_gateway/etc/kernel-launchers/scala/toree-launcher/target/scala-2.11/toree-launcher_2.11-2.3.0.SNAPSHOT.jar ...
[info] Done packaging.
[success] Total time: 13 s, completed Aug 3, 2020 12:02:17 PM
curl -L https://repository.apache.org/content/repositories/releases/org/apache/toree/toree-assembly/0.3.0-incubating/toree-assembly-0.3.0-incubating.jar --output ./kernel-launchers/scala/lib/toree-assembly-0.3.0-incubating.jar
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 27.9M  100 27.9M    0     0  6301k      0  0:00:04  0:00:04 --:--:-- 7191k
```

Reply:
Good catch, i had to update the make file and couple docker files as well. should be ok now.